ack := test.restore_of("func");
r := ack(2, 4);
if (r != 11)
    fail("restored ack() function gave incorrect result: " + string(r));
